packages/zephyr-core/zephyr-core.css

```diff
@@ -3,3 +3,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  @apply text-content dark:text-content-inverse;
```

Comment:
**Question:** I'm not sure how I feel about this. Should we consider using some sort of `<Text />` component instead of modifying any global styles?

Reply:
The `Text` component was actually my first approach and still not off the table but isn't there's the implicit problem where by default the font color for the app is black and therefore any text that's not accommodated will be invisible on dark mode.